### PR TITLE
Iterating on focus-ring documentation

### PR DESCRIPTION
**Pull request by heatherlarsen**
### :pushpin: Summary

If merged, this PR introduces changes to the focus ring documentation, including: 
- Making the content more actionable
- Hiding the showcase but keeping simple examples

I left a comment in the docs about the **Sass mixins** section...I'm unsure if what I've included is accurate, so I would love extra engineering eyes on that. 

**Preview link:** https://hds-website-git-heatherlarsen-foundations-focusring-hashicorp.vercel.app/foundations/focus-ring 

### :link: External links

Jira ticket: [HDS-1350](https://hashicorp.atlassian.net/browse/HDS-1350)

***

### 👀 Reviewer's checklist:

- [ ] +1 Percy if applicable
- [ ] Confirm that PR has a changelog update via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-1350]: https://hashicorp.atlassian.net/browse/HDS-1350?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/docs/foundations/focus-ring/index.md [hunk 1]
```diff
@@ -1,52 +1,59 @@
...
+While we offer two Sass mixins `hds-focus-ring-basic` and `hds-focus-ring-with-pseudo-element`, they’re mainly used in the design system codebase. 
 
-To use these mixins you have to import the Sass file `packages/components/app/styles/mixins/_focus-ring.scss` contained in the `@hashicorp/design-system` monorepo or the same file `app/styles/mixins/_focus-ring.scss` distributed in the `@hashicorp/design-system-components` package.
+In addition to applying the focus style, these mixins also account for all declarations of `:focus/:focus-visible` in the different browsers.
+
+If needing to use these mixins, import the Sass file: 
+
+<!-- IS THIS CORRECT? ARE THERE OTHER DETAILS WE NEED TO INCLUDE? -->
+```scss
+@import "~@hashicorp/design-system-components/app/styles/mixins/_focus-ring.scss";
```
Comment (didoo) [inline website/docs/foundations/focus-ring/index.md:53]: If I just copy&paste this, it is very likely that it will not work (I have to do other things, for the build process to reference that file correctly). That's why I kept it "generic".

**Review by jorytindall — APPROVED** [review]

File: website/docs/foundations/focus-ring/index.md (hunk 1 above)
Comment (heatherlarsen) [reply website/docs/foundations/focus-ring/index.md]: The way all of our Foundational documentation is written assumes that one has gone through and set things up appropriately based on the README or the new "Getting started for engineers" content. This was the case even before I made this change. I can include a link to the "Getting started" content on each page so that it's clear there are more steps involved before one gets to this point, but what was there originally was hard to follow and not very actionable, so I'm trying to improve it. 

I hoping to understand if this is the appropriate path and way to import this file into one's project. And if there are other steps related to **mixins** specifically that we need to include (as there is no documentation on mixins in the Getting started content). 

Happy to jump on a call to understand this process more too.

**Comment by heatherlarsen** [comment]
@didoo I've removed the mixins section per our discussion.

File: website/docs/foundations/focus-ring/partials/code/showcase.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <div class="dummy-focus-ring-sample">
+      <span class="dummy-text-small">critical</span>
+      <br />
+      <div class="hds-focus-ring-critical-box-shadow">
+        <Doc::Placeholder @text="with border radius" @width="100" @height="100" @background="transparent" />
+      </div>
+    </div>
+  </div>
+</section>
+-->
```
Comment (KristinLBradley) [inline website/docs/foundations/focus-ring/partials/code/showcase.md:30]: Is there a reason we want to comment this chunk out vs. just deleting it?

Comment (heatherlarsen) [reply website/docs/foundations/focus-ring/partials/code/showcase.md:30]: We don't want to delete it because we may want it in the future when we decide what to do with the showcase, but it doesn't need to be commented out since the reference to the showcase.md file is already commented out.

**Review by KristinLBradley — APPROVED** [review]
